website/docs/reference/datasources/querying-mongodb/README.md

```diff
@@ -245,7 +245,25 @@ people
 { $inc: { score: 1 } }
 ```
 
-`Limit`: The dropdown is used to configure if this delete command should act upon a single document or if this command should delete all the matching documents according to the query.
+`Limit`: The dropdown can be used to specify whether the command should apply to a single document or to all documents that match the query.
+
+
+MongoDB's multi update feature does not support replacement style updates. This means that you cannot replace the entire document, but rather, you can only update a single field.
+
+To successfully run a multi update command in MongoDB, you can use the following syntax:
+
+```js
+   { $set: { <field1>: <value1>, <field2>: <value2>, ... } },
```

Comment:
Can we also mention the field name in which this $set line has to be written ?

Reply:
I made some changes. Can you do a re-review?  Also, it's not clear to me about the mention of a field name. Could you please provide more context @sumitsum

Reply:
@harshilp24 I meant on Appsmith, where does the user write this text : `{$set ....}` - the name of that text field where the user would have to write it.

Reply:
updated :)